qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -954,14 +954,31 @@ def _build_qeom_result(
 
 
 class QEOMResult(EigensolverResult):
-    """The results class for the QEOM algorithm."""
+    """The results class for the qEOM algorithm.
+
+    Attributes:
+        ground_state_raw_result: The raw results of the ground state eigensolver.
+        excitation_energies: The excitation energies approximated by the qEOM algorithm.
+        expansion_coefficients: The expansion coefficients matrix of the excitation operators onto the
+            set of basis operators spanning the linear qEOM subspace.
+        eigenstates: Approximated excited states of the hamiltonian. Note that these are set to an empty
+            list as they cannot be provided as circuits by the qEOM algorithm but are required for the
+            results.
+        h_matrix: Matrix representing the Hamiltonian in the qEOM subspace.
+        s_matrix: Matrix representing the geometry of the qEOM subspace.
+        h_matrix_std: 2 by 2 matrix of the summed standard deviation of all elements of the matrices
+            M, Q and their conjugates.
+        s_matrix_std: 2 by 2 matrix of the summed standard deviation of all elements of the matrices
+            V, W and their conjugates.
```

Comment:
Referring to m, q, v, and w here does not work since these are not explained anywhere else. I would suggest we rather just refer to H and S, respectively, and properly define these somewhere here (probably in the few lines above explaining `h_matrix` and `s_matrix`.

Reply:
Done